### PR TITLE
Switch name from Story Squad to Scribble Stadium throughout the app

### PR DESCRIPTION
**Pull request by Mitch98k**
Why this was done: The stakeholders decided to change the name of this game in the app from Story Squad to Scribble Stadium. Story Squad will be the name of the app, which contains a game called Scribble Stadium.

What was done: Everywhere in the app where it says 'Story Squad' instead of 'Scribble Stadium', and is part of that particular game was changed to reflect the name change. In addition, any instance of that where the font family is not 'Bangers' was changed to that.

Trello card: https://trello.com/c/fvPE6JPl/510-story-squad-name-change

Loom video: https://www.loom.com/share/cbbf515028094c2f872fc537e2e221a5

*Correction on the video: The header file is for the gameplay component, not the parent dashboard.

### CODE REVIEW

**Review by artofmayhem — COMMENTED** [review]
on the test file check the lowercase scribble stadium. Should match the header text Scribble Stadium otherwise everything checks out and looks good. Nice to see the new name in action.

**Review by venegasgabby76 — DISMISSED** [review]
great job going though the app and replacing the name of the app to the appropriate one.

**Review by OniWarrior — DISMISSED** [review]
I've checked all the commits and there's no changes that I can see that would cause breaking changes. The only changes that were performed was the instance change from Story Squad to Scribble Stadium throughout the files that I checked in his commits. Everything is consistent with the exception of one commit in which the font family was changed in the gamemodestyles .less file from atma to bangers. I'm making the assumption this is just an aesthetic change to make the font look a little nicer. Other than that everything looks good and is ready to be merged.

**Review by ashtilawat — COMMENTED** [review]
Can you rebase main and push this branch up again. We changed gamification to gameplay everywhere in the app. If you rebase, it should fix the merge conflicts.

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Can you rebase main before I merge this

**Review by ashtilawat — DISMISSED** [review]

File: src/components/common/ParentNavTopBar.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const ParentNavTopBar = props => {
     <nav className="parent-nav-top-bar" theme="light">
       <Link to="/parent/dashboard">
         <Title className="title navbar-logo" level={1}>
-          STORY SQUAD
+          Scribble Stadium
```
Comment (just-kato) [inline src/components/common/ParentNavTopBar.js:42]: Good job with the name change

File: src/components/common/Header.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ const Header = ({
       {pointsRemaining && (
         <h2 className="points-remaining">POINTS REMAINING: {props.points}</h2>
       )}
-      <h1 className="header-text">{props.title || 'STORY SQUAD'}</h1>
+      <h1 className="header-text">{props.title || 'Scribble Stadium'}</h1>
```
Comment (just-kato) [inline src/components/common/Header.js]: Good Job

File: src/__tests__/Header.test.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const Component = () => {
 describe('<Header /> test suite', () => {
   it('the className for the header is correct', () => {
     const { getByText } = render(<Component />);
-    const head = getByText(/story squad/i);
+    const head = getByText(/scribble stadium/i);
```
Comment (just-kato) [inline src/__tests__/Header.test.js:27]: Good job

File: src/components/pages/GamePlay/GamePlayMain.js [hunk 4]
```diff
@@ -80,7 +80,7 @@ const GamemodeBtns = props => {
   // Enable initial modal
   useEffect(() => {
     const data = {
-      title: 'Welcome to StorySquad!',
+      title: 'Welcome to Scribble Stadium!',
```
Comment (just-kato) [inline src/components/pages/GamePlay/GamePlayMain.js:95]: Nice!

File: src/styles/less/GamemodeStyles.less [hunk 5]
```diff
@@ -334,7 +334,7 @@
         justify-content: center;
         text-align: center;
         flex-direction: column;
-        font-family: 'atma';
+        font-family: 'bangers';
```
Comment (just-kato) [inline src/styles/less/GamemodeStyles.less:337]: Nice font change

File: src/utils/helpers.js [hunk 6]
```diff
@@ -9,9 +9,9 @@ export function getBase64(file) {
 
 export const modalInstructions = {
   childDash:
-    'Welcome to Story Squad. Accept your mission to start an adventure!',
+    'Welcome to Scribble Stadium. Accept your mission to start an adventure!',
   missionControl1:
-    'Welcome to Story Squad! To begin your journey click the "READ" icon to start the story! Are you ready to accept the challenge?',
+    'Welcome to Scribble Stadium! To begin your journey click the "READ" icon to start the story! Are you ready to accept the challenge?',
```
Comment (just-kato) [inline src/utils/helpers.js:14]: Nice!

**Review by just-kato — DISMISSED** [review]

**Comment by bipin-shrestha** [comment]
Thanks Mitchell for taking the time to make the change throughout the app.

**Review by casey-dietz — DISMISSED** [review]
Looks great, nice updates! Great video going through all the changes and explaining what work was done. Nice work!

File: src/__tests__/Header.test.js (hunk 3 above)
Comment (just-kato) [inline src/__tests__/Header.test.js:27]: Nice name change!

File: src/components/pages/GamePlay/GamePlayMain.js [hunk 7]
```diff
@@ -92,7 +92,7 @@ const GamemodeBtns = props => {
...
```
Comment (just-kato) [inline src/components/pages/GamePlay/GamePlayMain.js:95]: Looks good

File: src/styles/less/GamemodeStyles.less (hunk 5 above)
Comment (just-kato) [inline src/styles/less/GamemodeStyles.less:337]: Nice font change

File: src/utils/helpers.js [hunk 8]
```diff
@@ -9,9 +9,9 @@ export function getBase64(file) {
 
 export const modalInstructions = {
   childDash:
-    'Welcome to Story Squad. Accept your mission to start an adventure!',
+    'Welcome to Scribble Stadium. Accept your mission to start an adventure!',
```
Comment (just-kato) [inline src/utils/helpers.js:12]: Little changes that matter! Great Job

File: src/utils/helpers.js [hunk 9]
```diff
@@ -39,10 +39,10 @@ export const getMissionControlText = (hasRead, hasDrawn, hasWritten) => {
 export const progressInfo = {
   welcome: 'Welcome to the Progress Page!',
   explanation:
-    "Below you will see some graphs representing what we at Story Squad call our 'Squad Score'. What exactly is a Squad Score? We're glad you asked!",
+    "Below you will see some graphs representing what we at Scribble Stadium call our 'Squad Score'. What exactly is a Squad Score? We're glad you asked!",
```
Comment (just-kato) [inline src/utils/helpers.js:42]: Great job

**Review by just-kato — APPROVED** [review]

**Review by OniWarrior — APPROVED** [review]

**Pull request merged**